website/static/js/pages/dashboard-page.js

```diff
@@ -40,6 +41,108 @@ request.fail(function(xhr, textStatus, error) {
     });
 });
 
+var confirmedEmailURL = window.contextVars.confirmedEmailURL;
+var removeConfirmedEmailURL = window.contextVars.removeConfirmedEmailURL;
+
+
+//choose which emails/users to add to an account and which to deny
+function confirm_emails(emails) {
+    if (emails.length > 0) {
+        var email = emails.splice(0,1);
```

Comment:
This line breaks removes the first element of `emails` (which is intended because this function recurses), but assigns `email` to be an array of length 1. Consider instead:
```javascript
var email = emails.splice(0,1)[0];
```